sprintmission7/schema.sql

```diff
@@ -0,0 +1,85 @@
+-- 유저
+CREATE TABLE users (
+    user_id SERIAL PRIMARY KEY,
+    email VARCHAR(100) NOT NULL,
+    nickname VARCHAR(50) NOT NULL,
+    password VARCHAR(100) NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    updated_at TIMESTAMP NOT NULL DEFAULT NOW()
+    );
+
+-- 상품 
+CREATE TABLE products (
+    product_id SERIAL PRIMARY KEY,
+    user_id INT,
+    product_name VARCHAR(50) NOT NULL,
+    product_description VARCHAR(150) NOT NULL,
+    price DECIMAL(10,2) NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    updated_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    FOREIGN KEY (user_id) REFERENCES users(user_id) ON DELETE CASCADE 
+  );
+
+-- 상품 좋아요
+CREATE TABLE product_like (
+    user_id INT,
+    product_id INT,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    PRIMARY KEY (user_id, product_id),
+    FOREIGN KEY (user_id) REFERENCES users(user_id) ON DELETE CASCADE,
+    FOREIGN KEY (product_id) REFERENCES products(product_id) ON DELETE CASCADE
+  );
+
+-- 게시글
+CREATE TABLE posts (
+    post_id SERIAL PRIMARY KEY,
+    user_id INT,
+    title VARCHAR(255) NOT NULL,
+    content TEXT NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    updated_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    FOREIGN KEY user_id REFERENCES users(user_id) ON DELETE CASCADE
```

Comment:
괄호가 누락되어 구문오류가 날 것 같네요. 수정 부탁드립니다.

```suggestion
    FOREIGN KEY (user_id) REFERENCES users(user_id) ON DELETE CASCADE
```